test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -65,6 +65,14 @@ def test_melspectrogram(self):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform], nondet_tol=1e-10)
 
+    def test_griffinlim(self):
```

Comment:
Could you add a case where momentum is disabled, `momentum=0`?

The in-place multiplication is causing the test failure.

https://github.com/pytorch/audio/blob/90311ed11ff88042b8ea5e849a746aedb10a0f42/torchaudio/functional/functional.py#L227